### PR TITLE
Update expectations in "Error message describes invalid form field value"

### LINKED ISSUE #1298: Error message describes invalid form field value (36b590): Undefined behaviour for form fields without error indicators
Each of the three expectations start with "For each test target with a form field error indicator, [...]" which made me realise during implementation of the rule that it's not currently defined what to do about the test targets that _don't_ have a form field error indicator. I therefore had to assume that they should simply pass.

In hindsight, it also seems a little odd to have the rule be applicable to form fields without error indicators, but that's currently an unfortunate consequence of not having an objective definition of what an error indicator is.

Summoning @carlosapaduarte, @joao-vicente, and @WilcoFiers.

### PR DESCRIPTION
**Pull request by carlosapaduarte**
Updated expectations to take into account form fields that do not have error indicators

Closes issue(s):
- closes #1298 

Need for Final Call:

This will require a 2 weeks Final Call

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Comment by WilcoFiers** [comment]
@carlosapaduarte can you give this a descriptive title though?

**Comment by carlosapaduarte** [comment]
> @carlosapaduarte can you give this a descriptive title though?

@WilcoFiers I will but I just need to make sure what does the "this" in your sentence refers to :-) Is it this PR?

**Comment by carlosapaduarte** [comment]
Final call ends June 19

File: _rules/invalid-form-field-value-36b590.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ in [text][] that is [visible][].
 
 ## Expectation 3
 
-For each test target with a [form field error indicator][], at least one of the [form field error indicators][form field error indicator] describes:
+Each test target either has no [form field error indicators][form field error indicator], or at least one of the [form field error indicators][form field error indicator] describes:
```
Comment (EmmaJP) [inline _rules/invalid-form-field-value-36b590.md:66]: Expectation 2 and 3 are identical ... either Expectation 3 is redundant or should describe something else.

**Comment by carlosapaduarte** [comment]
Final call ended. Merging.

**Pull request merged**